packages/api-elements/test/api-description-test.js

```diff
@@ -943,6 +1007,18 @@ describe('API description namespace', () => {
       });
     });
 
+    it('should have hosts', () => {
+      expect(transition.hosts.first.attributes.get('href').toValue()).to.deep.equal('https://first-server.com');
```

Comment:
```suggestion
      expect(transition.hosts.first.href.toValue()).to.deep.equal('https://first-server.com');
```

Would the `href` accessor on resource work here?

Reply:
changed